ProjectCoimbra.UWP/Project.Coimbra/Strings/en-US/Resources.resw

```diff
@@ -329,4 +329,48 @@
   <data name="LaneSetupControl.KeysSelected" xml:space="preserve">
     <value>Keys {0} selected.</value>
   </data>
-</root>
+  <data name="OptionsPage.Title.Text" xml:space="preserve">
+    <value>Well done! What would you like to do next?</value>
+    <comment>The text of the Title on the Options Page</comment>
+  </data>
+  <data name="OptionsPage.ChangeNoteDurationButton.Content" xml:space="preserve">
```

Comment:
These resources should also be added to the "de-DE" file, which currently is identical. I'd suggest just copying and pasting this file on top of the other.